make/autoconf/flags-cflags.m4

```diff
@@ -298,7 +298,9 @@ AC_DEFUN([FLAGS_SETUP_OPTIMIZATION],
       C_O_FLAG_NONE="${C_O_FLAG_NONE} ${DISABLE_FORTIFY_CFLAGS}"
     fi
   elif test "x$TOOLCHAIN_TYPE" = xclang; then
-    if test "x$OPENJDK_TARGET_OS" = xmacosx; then
+    # Use -Os on aarch64 to work around known llvm issue,
+    # (see https://bugs.llvm.org/show_bug.cgi?id=44581) which makes release build crash in aarch64.
+    if test "x$OPENJDK_TARGET_OS" = xmacosx || test "x$OPENJDK_TARGET_CPU" = xaarch64; then
```

Comment:
Would you consider contributing this change to jdk/jdk mainline?

Reply:
I suppose the reporter will do it.